gqlalchemy/reference/transformations/translators/nx_translator.md

```diff
@@ -0,0 +1,111 @@
+---
+sidebar_label: nx_translator
+title: gqlalchemy.transformations.translators.nx_translator
+---
+
+# Abstract
+
+Translates data from Memgraph to NX graph and vice-versa.
+Uses original ids from Memgraph. Labels are encoded as properties. Since Networkx allows that nodes have properties of different dimensionality, this modules makes use of it and stores properties as dictionary entries. All properties are saved to Networkx data structure.
```

Comment:
```suggestion
Uses original IDs from Memgraph. Labels are encoded as properties. Since Networkx allows nodes to have properties of different dimensionality, this module stores properties as dictionary entries. All properties are saved to the NetworkX data structure.
```